agent/README.md

```diff
@@ -0,0 +1,45 @@
+# OpenTelemetry Java Agent Example
+
+This example demonstrates simple usage of the OpenTelemetry Java Agent published
+by [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation).
+
+It consists of a spring boot application with:
+
+- A gradle task for downloading the OpenTelemetry Java Agent.
+- A simple web API available at `GET http://localhost:8080/ping`. When called,
+  auto instrumentation from the OpenTelemetry Java Agent records spans and
+  metrics. Additionally, there is manual trace, metric, and logging
+  instrumentation.
```

Comment:
Nit: WDYM by manual logging instrumentation?

Reply:
The app uses the log4j api to produce application logs in a similar way that it might use the otel trace or metric API to produce telemetry beyond what is provided by auto instrumentation. 

It does feel a bit odd to call it "manual".